README.md

```diff
@@ -32,6 +32,8 @@ $ cp .env.template .env  # (first time only) #"cp is not recognised as an intern
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+See UsingTrello.md for instruction on environment variables which must be added to `.env`.
```

Comment:
Minor, but it might be nice to make this a link to make it really easy!
```suggestion
See [UsingTrello.md](./todo_app/UsingTrello.md) for instruction on environment variables which must be added to `.env`.
```
Relatedly, it might be nice to move that markdown file into a `docs` folder or similar just to make it easier to navigate as we grow the docs